--- Monster-1.23-src/src/util.cpp~	2007-08-18 21:39:49.000000000 +0200
+++ Monster-1.23-src/src/util.cpp	2007-08-18 21:38:56.000000000 +0200
@@ -127,7 +127,7 @@
 		}
 	}
 	else {
-		strcpy(path, "data/");
+		strcpy(path, "/usr/share/monster/data/");
 	}
 
 	return path;
@@ -146,7 +146,7 @@
 		}
 	}
 	else {
-		strcpy(path, "data/");
+		strcpy(path, "/usr/share/monster/data/");
 	}
 
 	return path;
@@ -167,9 +167,6 @@
 
 char* getScriptExtension()
 {
-	if (config.useCompiledScripts())
-		return "scr";
-	else
 		return "lua";
 }
 
